docs/tutorials/using-pixy-camera.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Using the CMUcam5 Pixy for LEGO Mindstorms
+group: hardware-extras
+author: "Kees Smit"
```

Comment:
If you like, you can use your github user here instead. Then the page will pick up your picture and link to your github profile.

    author: "@KWSmit"